src/components/SearchBar/index.tsx

```diff
@@ -73,31 +74,52 @@ export default function SearchBar({
     setSearchText(suggestion)
   }
 
+  let searchResults = <div>No results</div>
+  if (gameList.length > 0) {
+    searchResults = (
+      <>
+        {gameList.map((el) => (
+          <button
+            onClick={() => handleOnClickSuggestion(el)}
+            key={el}
+            className={styles.searchResult}
+          >
+            {el}
+          </button>
+        ))}
+      </>
+    )
+  }
+
   return (
-    <div className={classNames(styles.searchBar, containerClass)}>
-      <button className={styles.searchButton}>
-        <MagnifyingGlass fill="var(--color-neutral-100)" />
-      </button>
-      <input
-        ref={input}
-        type="text"
-        placeholder={placeholder}
-        {...inputProps}
-      />
-      {showClearButton && (
-        <button className={styles.clearButton} onClick={clearSearch}>
-          <CloseButton fill="var(--color-neutral-100)" onClick={clearSearch} />
-        </button>
-      )}
-      {gameList.length > 0 && (
-        <ul className={styles.autoComplete}>
-          {gameList.map((el) => (
-            <li onClick={() => handleOnClickSuggestion(el)} key={el}>
-              {el}{' '}
-            </li>
-          ))}
-        </ul>
-      )}
-    </div>
+    <Popover
```

Comment:
Popover provides a lot of important ui improvements like dynamically sized dropdown based on the target size